### PR TITLE
feat(chromium): use bundled ffmpeg instead of npm deps

### LINKED ISSUE #3680: [health] bundle ffmpeg with Chromium
We need ffmpeg in Chromium to generate screencast. We'll have it in two steps:
- [x] short-term: using [@ffmpeg-installer/ffmpeg](https://www.npmjs.com/package/@ffmpeg-installer/ffmpeg) to experiment with screencast API
- [ ] medium-to-long-term: bundle ffmpeg with Chromium binary

We'll land the [short-term solution](https://github.com/microsoft/playwright/pull/3668) to unblock @yury-s now, and will work on bundling ffmpeg in Chromium.

### PR DESCRIPTION
**Pull request by aslushnikov**
Fixes #3680

### CODE REVIEW

File: src/utils/browserPaths.ts [hunk 1]
```diff
@@ -72,6 +72,18 @@ export function windowsExeAndDllDirectories(browserPath: string, browser: Browse
   return [];
 }
 
+export function ffmpegPath(browserPath: string, browser: BrowserDescriptor): string {
+  if (browser.name !== 'chromium')
+    return '';
+  if (os.platform() === 'linux')
+    return 'ffmpeg';
+  if (os.platform() === 'win32')
+    return path.join(browserPath, 'chrome-linux', 'ffmpeg');
```
Comment (mxschmitt) [inline src/utils/browserPaths.ts:81]: shouldn't it be ffmpeg.exe for windows?

File: package.json [hunk 2]
```diff
@@ -37,7 +37,6 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@ffmpeg-installer/ffmpeg": "^1.0.20",
     "debug": "^4.1.1",
```
Comment (mxschmitt) [inline package.json:40]: does package-lock.json not change?

**Review by mxschmitt — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (aslushnikov) [reply package.json:40]: indeed! re-generated

File: src/utils/browserPaths.ts (hunk 1 above)
Comment (aslushnikov) [reply src/utils/browserPaths.ts]: ah! time to sleep!

**Review by mxschmitt — APPROVED** [review]
nit: EOL in videoRecorder.ts changed. Otherwise LGTM!

**Comment by aslushnikov** [comment]
> nit: EOL in videoRecorder.ts changed. Otherwise LGTM!

Which is probably [for the good](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)! 🤷‍♂️

**Pull request merged**